x-pack/test_serverless/functional/test_suites/common/discover/esql/_esql_view.ts

```diff
@@ -0,0 +1,531 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+  const log = getService('log');
+  const dataGrid = getService('dataGrid');
+  const testSubjects = getService('testSubjects');
+  const monacoEditor = getService('monacoEditor');
+  const security = getService('security');
+  const inspector = getService('inspector');
+  const retry = getService('retry');
+  const browser = getService('browser');
+  const find = getService('find');
+  const esql = getService('esql');
+  const dashboardAddPanel = getService('dashboardAddPanel');
+  const PageObjects = getPageObjects([
+    'svlCommonPage',
+    'common',
+    'discover',
+    'dashboard',
+    'header',
+    'timePicker',
+    'unifiedFieldList',
+  ]);
+
+  const defaultSettings = {
+    defaultIndex: 'logstash-*',
+  };
+
+  describe('discover esql view', async function () {
+    before(async () => {
+      await kibanaServer.savedObjects.cleanStandardList();
+      await security.testUser.setRoles(['kibana_admin', 'test_logstash_reader']);
+      log.debug('load kibana index with default index pattern');
+      await kibanaServer.importExport.load('test/functional/fixtures/kbn_archiver/discover');
+      // and load a set of makelogs data
+      await esArchiver.loadIfNeeded('test/functional/fixtures/es_archiver/logstash_functional');
+      await kibanaServer.uiSettings.replace(defaultSettings);
+      await PageObjects.svlCommonPage.loginAsAdmin();
+      await PageObjects.common.navigateToApp('discover');
+      await PageObjects.timePicker.setDefaultAbsoluteRange();
+    });
+
+    describe('test', () => {
+      it('should render esql view correctly', async function () {
+        await PageObjects.unifiedFieldList.waitUntilSidebarHasLoaded();
+
+        expect(await testSubjects.exists('showQueryBarMenu')).to.be(true);
```

Comment:
Done https://github.com/elastic/kibana/pull/181570/commits/c54c0a4c43d2c8637a5213f1953a1274236956bc

Reply:
@stratoula I think we don't need the wrapping `expect` there. So instead of
```
expect(await testSubjects.existOrFail('showQueryBarMenu'));
```
it could be
```
await testSubjects.existOrFail('showQueryBarMenu');
```

Reply:
thanx done, at this point I think I should mention that I didnt create this test from scratch. I just copied the code from the stateful and adapted on the serverless envirnoment.